locales/it/index.yml

```diff
@@ -632,6 +632,13 @@ payment:
       iuv: Codice IUV {{iuv}}
 wallet:
   wallet: Pagamenti
+  accessibility:
+    cardsPreview: Anteprima dei metodi di pagamento. Attiva per accedere alla lista dei tuoi metodi
+    transactionListItem:
+      payment:
+        read: Pagamento effettuato
```

Comment:
Could it be
read: Transazione
unread: Transazione da leggere
Like we did [for messages](https://github.com/pagopa/io-app/blob/5db8cd13d107f55e8d9ba5e2959445f84207a6e8/locales/it/index.yml#L915)?

Reply:
Yes! This sounds better!